xtask/src/main.rs

```diff
@@ -36,46 +36,54 @@ fn main() {
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("expand").about("Expands r9 macros").args(&[
             clap::arg!(--release "Build release version").conflicts_with("debug"),
             clap::arg!(--debug "Build debug version (default)").conflicts_with("release"),
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("kasm").about("Emits r9 assembler").args(&[
             clap::arg!(--release "Build release version").conflicts_with("debug"),
             clap::arg!(--debug "Build debug version (default)").conflicts_with("release"),
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("dist").about("Builds a multibootable r9 image").args(&[
             clap::arg!(--release "Build a release version").conflicts_with("debug"),
             clap::arg!(--debug "Build a debug version").conflicts_with("release"),
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("test").about("Runs unit tests").args(&[
             clap::arg!(--release "Build a release version").conflicts_with("debug"),
             clap::arg!(--debug "Build a debug version").conflicts_with("release"),
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("clippy").about("Runs clippy").args(&[
             clap::arg!(--release "Build a release version").conflicts_with("debug"),
             clap::arg!(--debug "Build a debug version").conflicts_with("release"),
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("qemu").about("Run r9 under QEMU").args(&[
             clap::arg!(--release "Build a release version").conflicts_with("debug"),
             clap::arg!(--debug "Build a debug version").conflicts_with("release"),
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("qemukvm").about("Run r9 under QEMU with KVM").args(&[
             clap::arg!(--release "Build a release version").conflicts_with("debug"),
             clap::arg!(--debug "Build a debug version").conflicts_with("release"),
+            clap::arg!(--verbose "Print commands"),
         ]))
         .subcommand(clap::Command::new("clean").about("Cargo clean"))
         .get_matches();
     if let Err(e) = match matches.subcommand() {
-        Some(("build", m)) => build(build_type(m)),
-        Some(("expand", m)) => expand(build_type(m)),
-        Some(("kasm", m)) => kasm(build_type(m)),
-        Some(("dist", m)) => dist(build_type(m)),
-        Some(("test", m)) => test(build_type(m)),
-        Some(("clippy", m)) => clippy(build_type(m)),
-        Some(("qemu", m)) => run(build_type(m)),
-        Some(("qemukvm", m)) => accelrun(build_type(m)),
+        Some(("build", m)) => build(build_type(m), verbose(m)),
```

Comment:
At this point, it may make sense to create a `BuildParams` struct that contains the build type and verbosity bool, create that, and pass it to the various build funcs.

Reply:
yeah, i agree.  i was looking at that, can't remember why i went back to this.  i think i didn't want to make too many more changes in this PR.